.testdata/sample-versions.txt

```diff
@@ -0,0 +1,19 @@
+# WARNING: file generated by gnomes DO NOT EDIT
+#
+# These are versions which are passed to runtests, on each architecture, for CI
+# testing. These do not need to be authoritative.
+#
+# Listing all versions ever provides for quadratic test runtime as we expand
+# architectures and OSes. So we no longer do that.
+#
+# Instead we include the highest patch release from each of the three supported
+# minor versions plus the highest patch release from the two most recent
+# unsupported minor versions.
```

Comment:
Shouldn't `two` and `three` be swapped here? (Go upstream only supports two minor versions (plus any RCs) at a time :sweat_smile:


![image](https://user-images.githubusercontent.com/161631/197848455-492f99e7-ae6c-456d-8cbf-6db1904d3421.png)

Reply:
They *should* be swapped 😯 Shame on me for trusting my memory! 😂